core/driver/sources/src/main/resources/reference.conf

```diff
@@ -161,34 +201,108 @@ probe {
+        // and fail the test. By default, `includeMessages` contains one regular expression, that matches
+        // all IDE errors messages. So, any IDE error will make the test fail if `driver.check.errors.enabled`
+        // is set to `true`. If you want to include only a subset of errors whose substrings match certain
+        // `includeMessages` entries, you have to put them into this field.
+        //
+        // `driver.check.errors.excludeMessages` defines which IDE error messages should NOT make ide-probe
+        // throw an exception and fail the test. In other words, here you can define, which IDE errors
+        // should be ignored even if `driver.check.errors.enabled` is set to `true`.
+        //
+        // Both for `includeMessages` and for `excludeMessages` you can define an entry in one of two possible ways:
+        // 1. By using a regex-based string. For example: ".*at com.intellij.uast.UastMetaLanguage.*" will match
+        //    any error message that contains "at com.intellij.uast.UastMetaLanguage". Note: your regex-based string
+        //    CAN NOT be a multi-line string.
+        // 2. By using a raw string without regular expressions. In such case ide-probe just checks if IDE error message
+        //    contains your raw string. The benefit of this option is that you can use a multi-line string (part of
+        //    stacktrace). You don't have to worry about line indentations here as each line is trimmed before performing
+        //    the check. If you want to use a multi-line stirng, use triple quotes: """<your multi-line string...>""".
+        //
+        // If an IDE error matches both an entry from `includeMessages` and another entry from `excludeMessages`,
+        // then such IDE error will be ignored (`excludeMessages` takes precedence).
         includeMessages = [".*"]
         excludeMessages = []
       }
+
+      // `driver.check.freezes.enabled` defines if ide-probe should throw an exception and fail the test if IDE freezes
+      // have occured during test. By default, this functionality is disabled. Set to `true` to enable.
       freezes.enabled = false
     }
+
+    // `driver.display` defines the display mode which should be used for the IDE run. By default, it is "native",
+    // which means that everything is displayed on the screen and you can interact with the IDE during test.
+    // Another possible value is "xvfb" which makes the IDE run in the xvfb mode (no display). This config can be set
+    // either with a raw string value or by setting the IDEPROBE_DISPLAY environment variable which overrides the default value.
     display = "native" # OR "xvfb" - these are two possible values
     display = ${?IDEPROBE_DISPLAY}
+
+    // `driver.xvfb` contains parameters that are used if IDE runs in the xvfb mode. Basically, it contains three
+    // screen size parameters that are passed to the "−screen" command line switch of xvfb.
```

Comment:
Strictly speaking, they're passed to [`xvfb-run --server-args="..."`](https://manpages.ubuntu.com/manpages/xenial/man1/xvfb-run.1.html), which then subsequently passes them to `Xvfb`

Reply:
Coz `xvfb-run` runs two processes:
1. Xvfb (virtual display server)
2. the specified process, with X display config pointing to (1.)

Reply:
Ok, but if I get it right, `xvfb-run` is a _wrapper_ for the `Xvfb` (1x - for now it's version 11). And in my comments I added a link to the https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml docs. And in this manual, they say:
```
In addition to the normal server options described in the [Xserver(1)](https://www.x.org/releases/X11R7.6/doc/man/man1/Xserver.1.xhtml)
manual page, Xvfb accepts the following command line switches:
−screen screennum WxHxD

This option creates screen screennum and sets its width, height, and depth to W, H, and D respectively. By default, only screen 0 exists and has the dimensions 1280x1024x8.
...
```
So, I think we can say that these 3 params are passed to the "−screen" command line switch of Xvfb?

Reply:
BTW, just changed from "xvfb" to "Xvfb" on this line to be more precise